build-logic-commons/build-platform/build.gradle.kts

```diff
@@ -25,7 +25,7 @@ dependencies {
     constraints {
         api("org.gradle.guides:gradle-guides-plugin:0.23")
         api("org.apache.ant:ant:1.10.14") // Bump the version brought in transitively by gradle-guides-plugin
-        api("com.gradle:develocity-gradle-plugin:3.17.6") // Run `build-logic-settings/update-develocity-plugin-version.sh <new-version>` to update
+        api("com.gradle:develocity-gradle-plugin:3.18.1") // Run `build-logic-settings/update-develocity-plugin-version.sh <new-version>` to update
```

Comment:
🤔 The 8.10.1 patch release just went out, I think we shouldn't change the bundled version for 8.10 anymore. Let's do the PR on `master`.